instana/resource-synthetic-monitor_test.go

```diff
@@ -0,0 +1,204 @@
+package instana_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/stretchr/testify/require"
+
+	. "github.com/gessnerfl/terraform-provider-instana/instana"
+	"github.com/gessnerfl/terraform-provider-instana/instana/restapi"
+	"github.com/gessnerfl/terraform-provider-instana/testutils"
+)
+
+const syntheticMonitorTerraformTemplate = `
+resource "instana_synthetic_monitor" "example" {
```

Comment:
Would be good to include custom_properties as well

Reply:
Added.